flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -949,8 +949,6 @@ public void run() {
 						checkpointMetaData.getCheckpointId());
 				}
 			} catch (Exception e) {
-				e.printStackTrace();;
```

Comment:
Please squash with previous change or mark this as a fixup (`git ci --amend --fixup=HEAD^`)